server/index.ts

```
@@ -0,0 +1,945 @@
+    case 'p':
+      return `<p>${innerContent}</p>`;
+    case 'heading':
+    case 'h1':
+    case 'h2':
+    case 'h3':
+    case 'h4':
+    case 'h5':
+    case 'h6':
+      const level = element.getAttribute('level') ?? nodeName.charAt(1) ?? '1';
+      return `<h${level}>${innerContent}</h${level}>`;
+    case 'bulletList':
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)